alphabase/psm_reader/sage_reader.py

```diff
@@ -548,7 +548,7 @@ def _sage_spec_idx_from_scan_nr(scan_nr: str) -> int:
         The scan_nr field in Sage output.
 
     """
-    return int(scan_nr.split("=")[-1])
+    return int(scan_nr.split("=")[-1]) - 1
```

Comment:
looking at the test: I think there are more robust ways to go from 
`'controllerType=0 controllerNumber=1 scan=7846` to `7845` :-D